app/ide-desktop/lib/content/src/dashboard.css

```diff
@@ -384,6 +384,9 @@ video {
 .-mt-2 {
   margin-top: -0.5rem;
 }
+.-mt-4 {
```

Comment:
I was writing about it in other PR review. Please apply other PR reviews and ping me, I'm skipping reviewing this PR for now.

Reply:
Fixed.